hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -569,4 +661,25 @@ void apply(Table<?, DeletedBlocksTransaction> deleteTxnsTable,
         BatchOperation batch, long txnID, DeletedBlocksTransaction delTX)
         throws IOException;
   }
+
+  /**
+   * The SchemaHandler interface provides a contract for handling
+   * KeyValueContainerData and DeletedBlocksTransaction based
+   * on different schema versions.
+   */
+  public interface SchemaHandler {
+    void handle(KeyValueContainerData containerData,
+                DeletedBlocksTransaction tx) throws IOException;
```

Comment:
@xichen01 , please correct the indent here. It's 4 blank space indent for a wrapped line. Same for TestSchemaHandler#handle.